src/main/java/com/atlassian/db/replica/internal/aurora/AuroraReplicasDiscoverer.java

```diff
@@ -40,9 +38,7 @@ public List<AuroraJdbcUrl> fetchReplicasUrls(Connection connection) throws SQLEx
 
     private List<String> fetchReplicasServerIds(Connection connection) throws SQLException {
         List<String> ids = new LinkedList<>();
-        final String sql = ignoreInactiveReplicas ?
-            "SELECT server_id FROM aurora_replica_status() WHERE session_id != 'MASTER_SESSION_ID' and last_update_timestamp > NOW() - INTERVAL '5 minutes'" :
-            "SELECT server_id FROM aurora_replica_status() WHERE session_id != 'MASTER_SESSION_ID'";
+        final String sql = "SELECT server_id FROM aurora_replica_status() WHERE session_id != 'MASTER_SESSION_ID' and last_update_timestamp > NOW() - INTERVAL '5 minutes'";
```

Comment:
Have you run it on real Aurora ?

Reply:
Yes

Reply:
@wyrzyk Have you thought about extending builder to configure number of seconds/minutes instead having fixed 5 minutes?

Reply:
Adding an API is easy, but maintenance is always a cost. We should extend API only with a good use case.